### PR TITLE
Fix supg test results

### PR DESCRIPTION
**Pull request by gassmoeller**
This fixes a test result was wrongly merged in #3020. Not sure how both testers passed with the wrong results. Some PRs might need to be rebased to fix their tests.

### CODE REVIEW

File: tests/supg/screen-output [hunk 1]
```diff
@@ -8,7 +8,7 @@ Number of degrees of freedom: 948 (578+81+289)
    Postprocessing:
      RMS, max velocity:                  0.816 m/s, 1.37 m/s
      Temperature min/avg/max:            0 K, 0.1027 K, 1 K
-     Heat fluxes through boundary parts: -1.633 W, 0 W, -12.73 W, -144.7 W
+     Heat fluxes through boundary parts: -6.27e-08 W, 0 W, -4.888e-07 W, -5.556e-06 W
```
Comment (bangerth) [inline tests/supg/screen-output]: Hm, @tjhei had even flagged this!

**Comment by tjhei** [comment]
Crazy! Thanks for fixing.

**Comment by gassmoeller** [comment]
I think waiting for the testers for this one might take a while. The CIG tester just rescanned the repo and has a queue of 30 builds (takes 3 hours or so), and Timo's tester has a backlog of 6 or so. Maybe just merge this to make sure not too many broken branches are created?

**Comment by tjhei** [comment]
okay.

**Comment by gassmoeller** [comment]
Thanks

**Pull request merged**